web/packages/teleport/src/Roles/RoleEditor/Shared.tsx

```diff
@@ -25,13 +25,17 @@ import useTeleport from 'teleport/useTeleport';
 
 export const EditorSaveCancelButton = ({
   onSave,
+  onPreview,
   onCancel,
   disabled,
```

Comment:
Since we added `previewDisabled`, I think it's fitting now to rename `disabled` to `saveDisabled`. Otherwise, this may be interpreted as "disable all".